Resources/Locale/en-US/reagents/meta/medicine.ftl

```diff
@@ -88,6 +88,18 @@ reagent-desc-ultravasculine = Rapidly flushes toxins from the body, but places s
 reagent-name-ethylredoxrazine = ethylredoxrazine
 reagent-desc-ethylredoxrazine = Neutralises the effects of alcohol in the blood stream. Though it is commonly needed, it is rarely requested.
 
+reagent-name-healing-mixture = healing mixture
+reagent-desc-healing-mixture = A mixture of reagents that makes you heal.
+
+reagent-name-robust-healing-mixture = robust healing mixture
+reagent-desc-robust-healing-mixture = A mixture of reagents that makes you heal fast.
+
+reagent-name-damage-resist-mixture = damage resistant mixture
+reagent-desc-damage-resist-mixture = A mixture of reagents that makes you resistant to damage.
+
+reagent-name-radaway = radaway
+reagent-desc-radaway = An odd reagent that cures your radiation.
+=======
```

Comment:
This definitely isn't meant to be here. This is a line inserted by git conflicts, and is causing the yaml parser to reject it.